File: src/net/eduard/permissions/command/PermissionsGroupCommand.java

package net.eduard.permissions.command;

import net.eduard.api.lib.manager.CommandManager;

public class PermissionsGroupCommand extends CommandManager {

	public PermissionsGroupCommand() {
		super("group","grupo");
		setDescription("Controlar os grupos de permissões");
		register(new PermissionsGroupCreateCommand());
		register(new PermissionsGroupDeleteCommand());
		register(new PermissionsGroupRemovePermissionCommand());
		register(new PermissionsGroupAddPermissionCommand());
		register(new PermissionsGroupSetPrefixCommand());
		register(new PermissionsGroupSetSuffixCommand());
		register(new PermissionsGroupSetDefaultCommand());
		register(new PermissionsGroupHelpCommand());
		
	}
	

}

File: src/main/java/net/eduard/permissions/command/PermissionsGroupCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command

class PermissionsGroupCommand : Command("group", "grupo") {
    init {
        description = "Controlar os grupos de permissões"
        register(PermissionsGroupCreateCommand())
        register(PermissionsGroupDeleteCommand())
        register(PermissionsGroupRemovePermissionCommand())
        register(PermissionsGroupAddPermissionCommand())
        register(PermissionsGroupSetPrefixCommand())
        register(PermissionsGroupSetSuffixCommand())
        register(PermissionsGroupSetDefaultCommand())
        register(PermissionsGroupHelpCommand())
    }
}